### DISCUSSION TITLE
Certificate suspending

### DISCUSSION THREAD

**Discussion by talcher** [Keyfactor/ejbca-ce#354] [Q&A]
Hello the EJBCA team,

could you help me know how to suspend a certificate for some time, before re-activating it - or not -, with EJBCA?
I have read different pages over the Web regarding such a feature - even [this one](https://sourceforge.net/p/ejbca/discussion/123123/thread/88fef1ef/)... However, I am afraid the [documentation itself](https://doc.primekey.com/ejbca/search?q=suspend&max=10) might not help much on how to do it; though I may have failed in properly looking for the information.

Could anyone give me a hint on the matter?

Thanks in advance,

talcher

**Comment by talcher** [comment]
I think I have finally found what I was looking for: the "Certificate Hold" action, in the Web UI.
Sorry for the useless discussion...

**Comment by primetomas** [comment]
Correct. You can also rad about Certificate Hold in [RFC5280](https://datatracker.ietf.org/doc/html/rfc5280).
